packages/vuetify/src/components/VTabs/__tests__/VTab.spec.ts

```diff
@@ -89,28 +89,139 @@ describe('VTab.ts', () => {
     expect(toggle).toHaveBeenCalledTimes(1)
   })
 
-  it('should respond to clicks and mousedown.enter', () => {
-    const event = { preventDefault: jest.fn() }
-    const toggle = jest.fn()
+  it('should toggle on click, keydown.enter, and keydown.space', () => {
```

Comment:
This seems like a good candidate for https://jestjs.io/docs/en/api#testeachtablename-fn-timeout